draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -256,6 +255,24 @@ export default class MentionSuggestions extends Component {
     }
   };
 
+  renderSuggestion = (mention, index) => {
+    const { theme = {} } = this.props;
+    const key = genKey();
```

Comment:
If we generate the key for every entry on every render React might not complain, but we end up with more DOM manipulations. The point is to use a unique identifier as key so React can reuse structures already existing in the DOM. Read more here: https://facebook.github.io/react/docs/multiple-components.html#dynamic-children

In addition the mention option is important for accessibility. That's the reason why we need this id attribute. Please don't change that.

Ideally we have some kind of mechanism that creates a unique ID for each mention.